src/hiccup/compiler.clj

```diff
@@ -359,3 +389,10 @@
         step2 (fn [] (compile-multi `util/*escape-strings?* [true false] step1))
         step3 (fn [] (compile-multi `util/*html-mode* [:html :xhtml :xml :sgml] step2))]
     (step3)))
+
+(comment
+  (require 'hiccup2.core)
+  (hiccup2.core/html [:span "a"])
+  (hiccup2.core/html [:span nil])
+  (hiccup2.core/html [:xml {:styles {'a 1 :b 2}}])
+  )
```

Comment:
Commented out development code should be removed.

Reply:
37d66cda9d19581db42a4b0dcd9855d0030bd437